### PR TITLE
[Export] Support retraceability test on existing cases

### PR DESCRIPTION
**Pull request by andrewlee302**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #115413
* __->__ #115402

Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

    Ran 71 tests in 31.929s
    
    OK (expected failures=14)

### CODE REVIEW

File: test/export/test_export.py [hunk 1]
```diff
@@ -1532,7 +1548,9 @@ def forward(self, x):
         ).run(core_aten_ep.graph_module.code)
         self.assertTrue(torch.allclose(core_aten_ep(*inp), m(*inp)))
 
+    @testing.expectedFailureRetraceability
     @testing.expectedFailureNonStrict
+    @testing.expectedFailureRetraceability
```
Comment (angelayi) [inline test/export/test_export.py:1553]: delete?

**Review by tugsbayasgalan — APPROVED** [review]

**Comment by malfet** [comment]
It feels more like a landrace then no-signal (i.e. test_retraceability was added by 2nd PR on the stack, but probably it introduced some features which were not there to beging with...

**Comment by andrewlee302** [comment]
> Also, I have to say this is a suspicious test plan.

@malfet  I think the commit may get overridden by old summary. The local test is totally fine, like you mentioned

> On the other hand, I just run the test inside the container against commit base and it indeed passes:

**Comment by andrewlee302** [comment]
Update the commit due to land race with another change.

@pytorchbot merge

**Pull request closed without merging**